### PR TITLE
[Travis] Validating yarn packages on travis tests suite

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | kind of |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Adding [Yarn check](https://yarnpkg.com/en/docs/cli/check)

I've tried both commands but the second one is not verbose enough!

yarn check log

### CODE REVIEW

**Comment by lchrusciel** [comment]
@pamil should this go to 1.0 branch? Because of my previous merge there more commit then it is supposed to be.

**Comment by loic425** [comment]
@lchrusciel @pjedrzejewski 
This seems to be not enough. I've to improve travis installation... Have you an idea about the problem ?
capture d ecran 2017-09-18 a 15 56 49

**Comment by pamil** [comment]
Sorry, looks like @lchrusciel change the base branch. Anyway, this one should come to 1.0 branch (IMHO).

**Comment by venyii** [comment]
@loic425 try to include `application.sh` in your file after `common.lib.sh`. This does the node setup stuff
```
source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../bash/common.lib.sh"
source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../bash/application.sh"
```

File: etc/travis/suites/application/script/validate-yarn-packages [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../bash/common.lib.sh"
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../bash/application.lib.sh"
```
Comment (venyii) [inline etc/travis/suites/application/script/validate-yarn-packages]: `application.sh` without `lib.` :P

**Comment by loic425** [comment]
@venyii thx, it's now ok. 
@pamil @lchrusciel I will try a rebase tomorrow but not sure to manage :)

**Comment by pamil** [comment]
Thanks Loic! 🎉

**Pull request merged**